constants_manager/constants_manager.py

```diff
@@ -7,19 +7,15 @@
 
 class ConstantsManager():
     def __init__(self, config_file_name='constants.ini', constants_name='ENV'):
-        self.config_file_name = config_file_name
-        self.constants_name = constants_name
+        self.config = configparser.RawConfigParser()
+        self.config.optionxform = str
+        self.config.read(config_file_name)
+        self.environment = 'DEFAULT'
+        if constants_name in os.environ:
+            self.environment = os.environ[constants_name]
```

Comment:
Really? So, why did you implement it as getting environ name every getting the value?
I guess it's not necessary to check environment variable every getting the value, rather it should get it just only once...